lib/matplotlib/axes/_axes.py

```diff
@@ -1823,7 +1828,10 @@ def step(self, x, y, *args, **kwargs):
           If 'mid', the jumps in *y* occur half-way between the
           *x*-values.
 
-        Return value is a list of lines that were added.
+        Note
+        ----
+        Additional parameters are the same as those for :func:`~matplotlib.pyplot.plot`.
```

Comment:
This line is slightly too long for our code standard (max 79 character). Can you break it after "for"?